### PR TITLE
feat: account linking

### PR DESCRIPTION
**Pull request by usmangeek**
fixes https://github.com/KILTprotocol/ticket/issues/1844
This pull request includes account linking feature which will not work. Further implementations are required from tjwelde.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -1,127 +1,128 @@
...
-              href: 'https://t.me/KILTProtocolChat',
-            },
-            {
-              label: 'Twitter',
-              href: 'https://twitter.com/Kiltprotocol',
-            },
-            {
-              label: 'Clan KILT (Unofficial)',
-              href: 'https://clankilt.io',
+    title: 'KILT Documentation',
```
Comment (weichweich) [inline docusaurus.config.js:3]: I think this is only a style change? We normally indent with 2 spaces. Could you revert this change?

File: docs/chain/linking.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: Link DID
+---
+
+<head>
+  <script>window ||= new Object(); window.kilt = new Object()</script>
+</head>
+
+You may link your on-chain DID, with your new web3name, to any accounts you have in the Polkadot ecosystem.
+
+In this way, people with your Polkadot ecosystem account address will be able to see your DID and your web3 name and, vice versa, those with your web3name can see your public address. This is especially useful for validators and collators, who can provide a personalized name that makes them easily identifiable.
```
Comment (weichweich) [inline docs/chain/linking.mdx:12]: To make diffs and reviews easier readable we always have a single sentence per line. Markdown renders a single new line as a space. So this change wouldn't change the layout.

```suggestion
In this way, people with your Polkadot ecosystem account address will be able to see your DID and your web3 name and, vice versa, those with your web3name can see your public address.
This is especially useful for validators and collators, who can provide a personalized name that makes them easily identifiable.
```

File: sidebars.js [hunk 3]
```diff
@@ -1,34 +1,35 @@
...
-  ],
-  chain: [
-    {
-      type: 'autogenerated',
-      dirName: 'chain',
-    },
-  ],
-  sdk: [
-    'sdk/introduction',
+    apps: [
```
Comment (weichweich) [inline sidebars.js:2]: This is also only a style change?

File: src/components/AccountLinking.tsx [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+        onChange={handlePayerChange}
+        disabled={filteredAccounts.length === 0}
+        defaultValue=""
+      >
+        {filteredAccounts.map((account) => (
+          <MenuItem key={account.meta.name} value={account.address}>
+            {account.meta.name} ({account.meta.source})
+          </MenuItem>
+        ))}
+      </MUISelect>
```
Comment (weichweich) [inline src/components/AccountLinking.tsx:129]: I can't select my sporran accounts for payment. Might be a bug? You filter the accounts by genesis hash, so that only accounts for the Spiritnet show up? Maybe sporran doesn't support that?

File: src/pages/index.js [hunk 5]
```diff
@@ -8,94 +8,104 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
-      title: 'Whitepaper',
-      imageUrl: 'img/whitepaper_light.svg',
-      imageUrlDark: 'img/whitepaper_dark.svg',
-      link: 'https://www.kilt.io/wp-content/uploads/2020/01/KILT-White-Paper-v2020-Jan-15.pdf',
-      description: (
-        <>Read up on the theoretical concepts behind KILT Protocol.</>
-      ),
-    },
-  ],
+    [
```
Comment (weichweich) [inline src/pages/index.js:11]: that's also only style change?

**Review by weichweich — CHANGES_REQUESTED** [review]
Very nice PR. But I had problems using only sporran for linking accounts.

**Comment by ntn-x2** [comment]
@usmangeek is there any progress on this? Do you need further help/support?

**Comment by weichweich** [comment]
not needed anymore.
